Formula/ecflow-ui.rb

```diff
@@ -1,17 +1,17 @@
 class EcflowUi < Formula
   desc "User interface for client/server workflow package"
   homepage "https://confluence.ecmwf.int/display/ECFLOW"
-  url "https://confluence.ecmwf.int/download/attachments/8650755/ecFlow-5.8.4-Source.tar.gz"
-  sha256 "bc628556f8458c269a309e4c3b8d5a807fae7dfd415e27416fe9a3f544f88951"
+  url "https://confluence.ecmwf.int/download/attachments/8650755/ecFlow-5.9.0-Source.tar.gz"
+  sha256 "20c1f457a21aebf7c49f06a4709a6216c6a92ca1899caa2c27d1aab4df53cacd"
   license "Apache-2.0"
 
   bottle do
-    sha256                               arm64_monterey: "4efd58a212460acd12d6a63e6bed76c639dc0610e13fc9e51dc96c93795292fc"
-    sha256                               arm64_big_sur:  "956e6e7f868bfb1749649deeb92d2f8a9e2ed51221a2450d7766fc5ca66991da"
-    sha256                               monterey:       "95f6398877a20b073daddb32a49bb19a7d83293339b2251f4db31ea091281877"
-    sha256                               big_sur:        "725dca63579c61e63c7f09a08af579e9069565cadc15a49ecb2318fbb3019483"
-    sha256                               catalina:       "e592ad1ce1f86b8777420ebaccfdfd44d7d7518f2bec209fec629a704a474754"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "94d367ac3a213b88c7331e759b56a053207d06646799a137e69732d443ac9bd6"
+    sha256 arm64_monterey: "4efd58a212460acd12d6a63e6bed76c639dc0610e13fc9e51dc96c93795292fc"
+    sha256 arm64_big_sur:  "956e6e7f868bfb1749649deeb92d2f8a9e2ed51221a2450d7766fc5ca66991da"
+    sha256 monterey:       "95f6398877a20b073daddb32a49bb19a7d83293339b2251f4db31ea091281877"
+    sha256 big_sur:        "725dca63579c61e63c7f09a08af579e9069565cadc15a49ecb2318fbb3019483"
+    sha256 catalina:       "e592ad1ce1f86b8777420ebaccfdfd44d7d7518f2bec209fec629a704a474754"
+    sha256 x86_64_linux:   "94d367ac3a213b88c7331e759b56a053207d06646799a137e69732d443ac9bd6"
   end
```

Comment:
Why?

Reply:
Good question @carlocab - this change was originally in #112710, which didn't go through because of a problem with Qt on Linux (a system problem I believe, not an ecFlow issue) - so, seeing that another project that used Qt did not have "cellar: :any_skip_relocation, x86_64_linux:" I removed it in case it would help. It's most likely irrelevant though, and I can easily revert that if desired.

Reply:
Hi @carlocab , I reverted my change but still get this on Linux:
```
==> Verifying checksum for '991d24a17734de6c50b82a19a2d8a63acb5b6f8a1dd141abd3ab4e6e4ba37444--xcb-util-wm--0.4.2.x86_64_linux.bottle.tar.gz'
  Error: qt: no bottle available!
```
We're not worried about supporting Linux for this package, so I'm tempted to remove Linux support from it - would that be ok, or is there some obvious fix for this issue?

Reply:
QT will be build for Linux again eventually and then it'll start working again. Changing the bottle block isn't the answer because it'll be overwritten by CI.

Reply:
Thanks @SMillerDev - will there be a way for me to know when QT is working again, and is there a way for me to force a rebuild of this PR without just adding more commits?

Reply:
Except following all pull request conversations, no.
You can rebase the branch against HEAD.